### ISSUE TITLE
🐛 [Bug] resnet50 model giving error when given dynamic shape input while aten::resize is in fallback

### ISSUE THREAD

**Issue by apbose** [pytorch/TensorRT#1465]
##  Bug Description

resnet50 torch model gives an error when given dynamic shape and having resize in fallback mode
```
compiled_cpp_mod = _C.compile_graph(module._c, _parse_compile_spec(spec))
RuntimeError: Trying to create tensor with negative dimension -1: [-1, 3, 224, 224]
```

## To Reproduce

1. Run torch_tensorrt.compile with resnet50 model as input, using fp32 precision and fp32 precision input
2. Put aten::resize in torch_executed_ops
3. Input shapes - shapes = {"input_ids": {"min": (1, 3, 224, 224),\
          "opt": (256, 3, 224, 224), "max": (256, 3, 224,224)}} with max_workspace_size = 8589934592

The above fails for maxine_vfx_MLP_expression as well. For the model, you could mail me.

## Environment
Torch-TensorRT Version: 1.3.0a0+e3b99294
PyTorch Version: 1.13.0.dev20220921+cu116
CPU Architecture: Intel Xeon CPU
OS: Ubuntu 18.03
How you installed PyTorch: pip
Build command you used: python setup.py develop
Are you using local sources or building from archives: local
Python version: 3.8.13
CUDA version: 11.6

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version (e.g. 1.0.0):
 - PyTorch Version (e.g. 1.0):
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by peri044** [comment]
I'm unable to reproduce this error. Were you trying with master branch ?

**Comment by apbose** [comment]
Yes this was in the master branch with dynamic shape fallback enabled

**Comment by peri044** [comment]
Can you post the entire script which has the reproducer code ?

**Comment by apbose** [comment]
`import pathlib
from typing import Optional
from unicodedata import name
import numpy as np
import torch_tensorrt
import torch


exported_model_path = "/home/abose/Documents/work/models_exported/models_Bermuda/resnet50-fp16-pytorch/NVIDIA-A100-80GB.nav/torchscript-trace/model.pt"
converted_model_path = "/home/abose/Documents/work/models_exported/models_Bermuda/resnet50-fp16-pytorch/NVIDIA-A100-80GB.nav/torch-trt-trace-fp32/model.pt"


shapes = {"input_ids": {"min": (1, 3, 224, 224),\
          "opt": (256, 3, 224, 224), "max": (256, 3, 224,224)}}
input_dtype = [torch.float32]
max_workspace_size = 8589934592

target_device = "cuda"
enabled_precisions = {'enabled_precisions': [torch.float32]}

# Adapted from convert function from reproduce_conversion.py
workdir = pathlib.Path.cwd()

model_input_shapes = []


for input_shapes, input_dtype in zip(shapes.values(), input_dtype):
    model_input_shapes.append(torch_tensorrt.Input( min_shape=input_shapes["min"],
                                                    opt_shape=input_shapes["opt"],
                                                    max_shape=input_shapes["max"],
                                                    dtype=input_dtype,
                                                  )
                              )
enabled_precisions = {'enabled_precisions': [torch.float32]}

exported_model_path = pathlib.Path(exported_model_path)
if not exported_model_path.is_absolute():
    exported_model_path = workdir

model = torch.jit.load(exported_model_path.as_posix(), map_location=target_device)

with torch_tensorrt.logging.debug():
    tr_model_compiled = torch_tensorrt.compile(module=model,
                                            inputs=model_input_shapes,
                                            workspace_size=max_workspace_size,
                                            truncate_long_and_double=True,
                                            torch_executed_ops=["aten::resize"],
                                            **enabled_precisions
                                            )

converted_model_path = pathlib.Path(converted_model_path)
if not converted_model_path.is_absolute():
    converted_model_path = workdir / converted_model_path
tr_model_compiled.save(converted_model_path.as_posix())`

**Comment by Christina-Young-NVIDIA** [comment]
Apurba to run with latest changes and post logs.

**Comment by apbose** [comment]
Posting the logs. The test passes with no dynamic shape. With the latest master changes, the test fails with
ERROR: [Torch-TensorRT TorchScript Conversion Context] - 4: [graphShapeAnalyzer.cpp::analyzeShapes::1294] Error Code 4: Miscellaneous (IShuffleLayer %453 : Tensor = aten::reshape(%x.1, %451): reshape dimension with more than one -1 wildcard. Reshaping [(# 0 (SHAPE input_0)),2048,1,1] to [-1,-1].)
 
[out.log](https://github.com/pytorch/TensorRT/files/10197100/out.log)

**Comment by Christina-Young-NVIDIA** [comment]
This was fixed by feature we released in 1.3.

**Comment by peri044** [comment]
Closing this issue since it's resolved. @apbose  Please open a new issue if you have any with the model.
